coral-component-colorinput/src/scripts/ColorInputSwatch.js

```diff
@@ -107,10 +109,12 @@ const ColorInputSwatch = Decorator(class extends BaseColorInputAbstractSubview(B
     if (cssColorValue) {
       this._elements.colorButton.style.backgroundColor = cssColorValue;
       this._elements.colorButton.label.textContent = hexColorValue;
+      this._elements.colorLabel.textContent = hexColorValue;
```

Comment:
Using `coral-tooltip` we set the `textContent` of the `content` element.

```suggestion
      this._elements.colorLabel.content.textContent = hexColorValue;
```